rtCamp/nginx-helper
# Fix _load_textdomain_just_in_time notice in WordPress 6.7

## Description
The aim of this PR is to fix the warning in WordPress 6.7 since the i18n functions were being invoked before the init hook was fired.

## Type
- Bug Fix

## Reference Issue
- https://github.com/rtCamp/nginx-helper/issues/364

## Testing
- Upgrade your WordPress to version 6.7
- Open the Nginx Helper Options Page.
- Install a non-English language translation for the plugin.
- Set the site language as the one selected above.
- Once you are on admin dashboard, ensure that there are no warnings displayed.

Example to install a non-English Language:

`wp language plugin install nginx-helper ru_RU` // Install the Russian Language translation.
`wp user meta update 1 locale ru_RU` // Change the current user language to russian.
